raven/utilities/mk_test.py

```diff
@@ -0,0 +1,202 @@
+# -*- coding: utf-8 -*-
+"""
+Created on Wed Jul 29 09:16:06 2015
+@author: Michael Schramm
+"""
+
+from __future__ import division
```

Comment:
This shouldn't be needed as python3 already converts integer-division with remainders to floats.

Reply:
Removed, thanks